infra/lib/common.ts

```diff
@@ -14,14 +14,20 @@ import * as secretsmanager from 'aws-cdk-lib/aws-secretsmanager';
 import * as wafv2 from "aws-cdk-lib/aws-wafv2";
 import * as triggers from 'aws-cdk-lib/triggers';
 import { Construct } from "constructs";
-import { StorageAccessLevel } from "./storage/stack";
+
 
 // TODO(dnys1): Remove when resources are consolidated in the same region again.
 export const env: cdk.Environment = {
   account: process.env.CDK_DEFAULT_ACCOUNT,
   region: 'us-west-2',
 };
 
+export enum StorageAccessLevel {
+  public = "guest",
+  protected = "protected",
+  private = "private",
+}
+
```

Comment:
I'm not sure if we need this anymore, we deleted the storage stack and hence we can remove storage config and also not adding it to the AmplifyConfig. we can also remove the analytic config and not add it to AmplifyConfig

Reply:
Good call out, I pushed a update removing the extra code.